### PR TITLE
Hook unit tests

### PR DESCRIPTION
**Pull request by muselesscreator**
Adds testing coverage for the hooks directory.

### CODE REVIEW

File: src/components/ModalActions/hooks/useFinishedStateActions.js [hunk 1]
```diff
@@ -61,6 +61,7 @@ const useFinishedStateActions = () => {
     // and there are no more assessments available for the current step
     return { primary: exitAction };
   }
+  console.log({ primary: startStepAction, secondary: exitAction });
```
Comment (leangseu-edx) [inline src/components/ModalActions/hooks/useFinishedStateActions.js:64]:
```suggestion
```

File: src/components/StatusAlert/index.jsx [hunk 2]
```diff
@@ -25,11 +27,13 @@ const StatusAlert = ({
   );
 
   React.useEffect(() => {
-    document.querySelector('html').scrollTo({
-      top: 0,
-      behavior: 'smooth',
-    });
-  }, [hasSubmitted, step]);
+    if (isMounted.current) {
```
Comment (leangseu-edx) [inline src/components/StatusAlert/index.jsx:30]: isn't `useIsMounted` here redundant?

File: src/data/services/lms/hooks/selectors/index.ts [hunk 3]
```diff
@@ -130,10 +130,12 @@ export const useTextResponses = () => {
...
 
-export default StrictDict({
+const exported = StrictDict({
   ...selectors,
   useStepState,
   useXBlockState,
   useActiveStepConfig,
   useGlobalState,
 });
+console.log({ exportedSelectors: exported });
```
Comment (leangseu-edx) [inline src/data/services/lms/hooks/selectors/index.ts:140]:
```suggestion
```

**Review by leangseu-edx — APPROVED** [review]

File: src/components/StatusAlert/index.jsx (hunk 2 above)
Comment (muselesscreator) [reply src/components/StatusAlert/index.jsx:30]: goal here is checking if *still* mounted, not if mounted *yet*

**Review by leangseu-edx — APPROVED** [review]

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @muselesscreator! Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.


:warning: We can't start reviewing your pull request until you've submitted a     [signed contributor agreement](https://openedx.org/cla)     or indicated your institutional affiliation. Please see the [CONTRIBUTING](https://github.com/openedx/.github/blob/master/CONTRIBUTING.md)     file for more information.     If you've signed an agreement in the past, you may need to re-sign.  See     [The New Home of the Open edX Codebase](https://open.edx.org/blog/the-new-home-of-the-open-edx-codebase/)     for details.

    
Once you've signed the CLA, please allow 1 business day for it to be processed.  After this time, you can re-run the     CLA check by adding a comment here that you have signed it.  If the problem persists, you can tag the `@openedx/cla-problems` team in a     comment on your PR for further assistance.

**Pull request merged**